### ISSUE TITLE
fast playback on resumed playback

### ISSUE THREAD

**Issue by markifi** [ferreum/mpv-skipsilence#2]
it works amazingly well, the only very slight thing i found is if mpv.conf has
`save-position-on-quit`
and you quit while it's sped up, open the file later to resume, playback speed defaults to the multiplied speed.

also the default threshold setting was low to me and i increased it to -26 dB, but i guess that's personal preference.

**Comment by ferreum** [comment]
Yeah, I'm aware of the problem, but I don't know if there's a way to prevent this. For now you'll have to reset the speed manually after resuming from saved position.

Alternatively you may prefer removing speed from `watch-later-options`, so it's not saved at all, by adding this to mpv.conf:
```
watch-later-options-remove=speed
```

**Comment by idMysteries** [comment]
> but I don't know if there's a way to prevent this.

https://[mpv.io](https://mpv.io/manual/master/#command-interface-shutdown)/manual/master/#command-interface-shutdown

If I understood correctly, then this event is just before the exit. Set the speed to base_speed.

**Comment by idMysteries** [comment]
```lua
local watch_later_list = mp.get_property("watch-later-options", {})
local watch_later_speed = mp.get_property_bool("options/save-position-on-quit") and watch_later_list:find("speed")

local function toggle(flag)
    local arg
    if flag == "no-osd" then
        arg = "no-osd"
    end
    if is_enabled then
        disable(arg)
        if watch_later_speed then
            mp.commandv("change-list", "watch-later-options", "append", "speed")
        end
    else
        enable(arg)
        if watch_later_speed then
            mp.commandv("change-list", "watch-later-options", "remove", "speed")
        end
    end
end
```

Something like that?
